src/java/com/zimbra/cs/nginx/NginxLookupExtension.java

```diff
@@ -111,14 +111,15 @@ public String getName() {
     public void init() throws ExtensionException, ServiceException {
         ExtensionDispatcherServlet.register(this, new NginxLookupHandler());
         CacheExtension.register("reverseproxylookup", new ReverseProxyCache());
-
+/*
```

Comment:
Please add a comment that these lines should be re-instated when https://jira.corp.synacor.com/browse/ZCS-1377 is fixed and update that Jira to say the same thing.

Reply:
reverted this section entirely.